src/test/java/kitchenpos/bo/TableGroupBoTest.java

```diff
@@ -0,0 +1,164 @@
+package kitchenpos.bo;
+
+import kitchenpos.dao.OrderDao;
+import kitchenpos.dao.OrderTableDao;
+import kitchenpos.dao.TableGroupDao;
+import kitchenpos.model.OrderTable;
+import kitchenpos.model.TableGroup;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class TableGroupBoTest {
+    @Mock
+    private TableGroupDao tableGroupDao;
+
+    @Mock
+    private OrderTableDao orderTableDao;
+
+    @Mock
+    private OrderDao orderDao;
+
+    @InjectMocks
+    private TableGroupBo tableGroupBo;
+
+    private TableGroup tableGroup;
+    private List<OrderTable> orderTableList;
+    OrderTable orderTable1;
```

Comment:
인스턴스 변수의 접근 제한자는 private으로 구현합니다.

```suggestion
    private OrderTable orderTable1;
```